src/main/kotlin/com/gamingbacklog/api/gamingbacklogapi/controllers/LibraryController.kt

```diff
@@ -60,6 +62,18 @@ class LibraryController(private val libraryService: LibraryService) {
     }
   }
 
+  @PostMapping("/games")
+  fun addToLibrary(
+    @RequestBody addGameToLibrary: UpdateLibraryGamesRequest,
+  ): ResponseEntity<List<LibraryResult>> {
```

Comment:
Don't we have this already? https://github.com/trauman413/gaming-backlog-api-kotlin/blob/d269d2436c8add8ea3bc8069cbfa93b2ae028611/src/main/kotlin/com/gamingbacklog/api/gamingbacklogapi/controllers/LibraryController.kt#L51
If this is for adding one game to multiple libraries could we update the name?